packages/node_modules/@ciscospark/spark-core/src/config.js

```diff
@@ -13,7 +13,7 @@ export default {
   maxAuthenticationReplays: 1,
   maxReconnectAttempts: 1,
   onBeforeLogout: [],
-  trackingIdPrefix: 'spark-js-sdk',
+  trackingIdPrefix: '',
```

Comment:
Could we add a quick comment that the `spark-js-sdk` string will be forced in ahead of this "prefix"?

Reply:
Done